common/gobuild.py.launcher.go

```diff
@@ -89,12 +91,29 @@ func main() {
 			}
 		}
 		// get payload if not empty
-		var payload map[string]interface{}
+		isJsonObjectParam := true
+		var payloadForJsonObject map[string]interface{}
+		var payloadForJsonArray []interface{}
 		if value, ok := input["value"].(map[string]interface{}); ok {
-			payload = value
+			payloadForJsonObject = value
+		} else {
+			if value, ok := input["value"].([]interface{}); ok {
+				payloadForJsonArray = value
+				isJsonObjectParam = false
+			}
 		}
 		// process the request
-		result := action(payload)
+		var result interface{}
+		funcMain := reflect.ValueOf(Main)
```

Comment:
I am not that familiar with the runtime-go code.
Where this `Main` comes from?

Reply:
It is the user's action's Main. i debugged the /init(or /run) before, when execute `/init`, it will inject user action codes to action container's suitable directory, the common/gobuild.py.launcher.go can recognize this Main method.